examples/CMakeLists.txt

```diff
@@ -174,7 +175,16 @@ if (UNIX)
     "${CMAKE_CURRENT_BINARY_DIR}/${TEST_NAME}_fail.xml"
     @ONLY)
   set(_env_vars)
-  list(APPEND _env_vars "PKG_CONFIG_PATH=${example_INSTALL_DIR}/lib/pkgconfig:${FAKE_INSTALL_PREFIX}/lib/pkgconfig:$PKG_CONFIG_PATH")
+  # On Debian and if the install prefix is /usr, the default CMAKE_INSTALL_LIBDIR of this project
+  # as set by GNUInstallDirs will be different from the one of the example project, 
+  # so let's hardcode it in that case
+  if(EXISTS "/etc/debian_version" AND "${CMAKE_INSTALL_PREFIX}" MATCHES "^/usr/?$")
```

Comment:
If we want to check that the prefix is `/usr`, should we remove the `?`?

According to the [CMake Regex Specification](https://cmake.org/cmake/help/latest/command/string.html#regex-specification), `?` is defined as `Matches preceding pattern zero or once only`. So, couldn't this mean that there would be zero matches, resulting in the `if` being `true` when it should really be `false`?

Reply:
I guess that the `?` is meant to match both `^/usr$` and `^/usr/$`. Just as a reference, this logic was taken from the relevant code in CMake itself: https://gitlab.kitware.com/cmake/cmake/-/blob/v3.20.0/Modules/GNUInstallDirs.cmake#L242 .